src/plugins/components/form/form-elements.js

```diff
@@ -33,19 +33,19 @@ module.exports = function () {
       '.textarea': {
         height: '16rem',
         display: 'block',
-        borderColor: theme('color.foggy-dark'),
+        borderColor: theme('colors.black.20'),
```

Comment:
Kerri mentioned that the decanter form elements needed to be updated to match My Account. So I'm having it match the My Account styles.